catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/filter/impl/SortByImpl.java

```diff
@@ -67,4 +67,14 @@ public PropertyName getPropertyName() {
   public SortOrder getSortOrder() {
     return sortOrder;
   }
+
+  @Override
+  public String toString() {
+    StringBuilder sb = new StringBuilder();
+    sb.append("propertyName=");
```

Comment:
✏️ I've seen a lot of toStrings that start with the implementation name and then curly brackets (with the fields and values inside of the curly's), could we add that in? EG, `SortByImpl{propertyName=val1...}`. 
![image](https://user-images.githubusercontent.com/9013416/52573023-64dd8080-2dd6-11e9-86bb-8af6920a0bdf.png)

I think its also the default intelliJ auto toString format as well which is a plus.

Reply:
Updated to use generated toString from IJ which matches your review comments